lib/DxilDia/DxcPixTypes.cpp

```diff
@@ -275,7 +275,7 @@ STDMETHODIMP dxil_debug_info::DxcPixStructType::GetFieldByName(
     _In_ LPCWSTR lpName,
     _Outptr_result_z_ IDxcPixStructField **ppField)
 {
-  std::string name = CW2A(lpName);
+  std::string name = CW2A(lpName).m_psz;
```

Comment:
For these string-related ones, I'm pretty sure the conversion will be allowed if you do `std::string name(CW2A(lpName));`, and that's better code anyway.